src/Microsoft.AspNet.OData.Shared/IEdmChangedObjectOfT.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+namespace Microsoft.AspNet.OData
+{
+    /// <summary>
+    /// Base interface to be implemented by any Delta object required to be part of the DeltaFeed Payload.
+    /// </summary>
+    /// <typeparam name="TStructuralType">Generic Type for changed object</typeparam>
+    public interface IEdmChangedObject<TStructuralType> : IEdmChangedObject
```

Comment:
>IEdmChangedObject [](start = 21, length = 17)

I think we don't need to prefix typed scenario using "Edm".

Same as Delta<T>,  we can just name it as "IChanged<T>"